### PR TITLE
Support PHP 8.0

### PR DESCRIPTION
**Pull request by vincentchalamon**

### CODE REVIEW

**Review by dunglas — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ php:
   - 7.2
   - 7.3
   - 7.4
+  - 8.0snapshot
```
Comment (Nek-) [inline .travis.yml:13]: I guess it could be without `snapshot` :) .

Comment (vincentchalamon) [reply .travis.yml]: Looks like travis doesn't support PHP 8.0 yet
> rbenv: version `8.0' not installed

**Comment by phil-davis** [comment]
PHP 8.0 on Travis is working now.
Can someone merge this, and make a release so that people who have this as a dependency can also move forward to PHP 8.0?

**Comment by nicodmf** [comment]
@dunglas do you have any reason not to merge this PR ?

**Comment by dunglas** [comment]
@nicodmf I'm not a maintainer of this project, I don't have the merge permission.

**Comment by phil-davis** [comment]
@pamil I see that you merged a PR last year. Can you please review this one?
I guess that you have maintainer privs here?

**Comment by Vasary** [comment]
Please merge this

**Review by pamil — APPROVED** [review]

**Comment by pamil** [comment]
Thanks, Vincent! :tada:

**Pull request merged**